Dockerfile

```
@@ -0,0 +1,16 @@
+FROM python:3.12-rc-alpine as base
+RUN apk update
+RUN apk add curl
+RUN curl -sSL https://install.python-poetry.org | python3 -
+ENV PATH="${PATH}:/root/.local/bin"
+WORKDIR /todo_app
+COPY pyproject.toml poetry.lock ./
+RUN poetry install 
+EXPOSE 5000
+COPY todo_app ./todo_app
+FROM base as production
```

Minor, but a couple of small line breaks just to separate out the different sections would probably make this easier to read, e.g. separating the stages:
```suggestion
COPY todo_app ./todo_app

FROM base as production
```